experimenter/experimenter/nimbus_ui/static/webpack.config.js

```diff
@@ -10,11 +11,24 @@ module.exports = {
     edit_branches: "./js/edit_branches.js",
     experiment_detail: "./js/experiment_detail.js",
     branch_detail: "./js/branch_detail.js",
+    features_page: "./js/features_page.js",
   },
   output: {
     filename: "[name].bundle.js",
     path: path.resolve(__dirname, "dist"),
   },
+  resolve: {
+    alias: {
+      "@codemirror/state": path.resolve(nodeModulesPath, "@codemirror/state"),
+      "@codemirror/view": path.resolve(nodeModulesPath, "@codemirror/view"),
+      "@codemirror/merge": path.resolve(nodeModulesPath, "@codemirror/merge"),
+      "@codemirror/lang-json": path.resolve(
+        nodeModulesPath,
+        "@codemirror/lang-json",
+      ),
+      codemirror: path.resolve(nodeModulesPath, "codemirror"),
+    },
+  },
```

Comment:
Can you remove these aliases and try adding direct dependencies for `@codemirror/state`, `@codemirror/view`, and `@codemirror-lang-json` in our package.json ?

Reply:
(at the same version as code mirror)